chroma-manager/requirements.dev

```diff
@@ -6,10 +6,10 @@
 # http://wiki.whamcloud.com/display/HYD/Adding+and+Updating+IML+Requirements
 behave>=1.1.0
 django-extensions
-django-nose
+django-nose==1.2
 django-supervisor==0.3.0
 fudge
-log4tailer
+https://bitbucket.org/jordilin/alerta/downloads/log4tailer-3.0.9.tar.gz
```

Comment:
is this an appropriate place to be pulling this package from, can this not be hosted on github?